is-with-analytics/simple/deployment.yaml

```diff
@@ -0,0 +1,336 @@
+      targetPort: 9443
+      protocol: TCP
+
+---
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: wso2is-with-analytics-is-analytics-service
+spec:
+  selector:
+    node: wso2is-with-analytics-is-analytics
+  ports:
+    -
+      name: 'thrift'
+      port: 9090
+      protocol: TCP
+    -
+      name: 'thrift-ssl'
+      port: 9091
+      protocol: TCP
+    -
+      name: 'spark-port-1'
+      port: 9543
+      protocol: TCP
+    -
+      name: 'spark-port-2'
+      port: 9711
+      protocol: TCP
+    -
+      name: 'spark-port-3'
+      port: 9611
+      protocol: TCP
+    -
+      name: 'spark-port-4'
+      protocol: TCP
+      port: 7712
+    -
+      name: 'spark-port-5'
+      protocol: TCP
+      port: 7612
+    -
+      name: 'spark-port-6'
+      protocol: TCP
+      port: 7070
+    -
+      name: 'spark-port-7'
+      protocol: TCP
+      port: 7443
+
+---
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: wso2is-with-analytics-is-dashboard-service
+  labels:
+    deployment: wso2is-with-analytics-dashboard
+spec:
+  ports:
+    # ports that this service should serve on
+    -
+      name: 'https'
+      port: 9643
+      protocol: TCP
+
+  # label keys and values that must match in order to receive traffic for this service
+  selector:
+    deployment: wso2is-with-analytics-dashboard
+
+---
+
+#deploying wso2 identity server and analytics
+
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: wso2is-with-analytics-is-deployment
+spec:
+  replicas: 1
+  minReadySeconds: 30
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        deployment: wso2is-with-analytics-is
+    spec:
+      hostAliases:
+        - ip: "127.0.0.1"
+          hostnames:
+            - "wso2is"
+      containers:
+        - name: wso2is-with-analytics-is
+          image: docker.wso2.com/wso2is:5.7.0
+          livenessProbe:
+            exec:
+              command:
+                - /bin/sh
+                - -c
+                - nc -z localhost 9443
+            initialDelaySeconds: 250
+            periodSeconds: 10
+          readinessProbe:
+            exec:
+              command:
+                - /bin/sh
+                - -c
+                - nc -z localhost 9443
+            initialDelaySeconds: 250
+            periodSeconds: 10
+          lifecycle:
+            preStop:
+              exec:
+                command:  ['sh', '-c', '${WSO2_SERVER_HOME}/bin/wso2server.sh stop']
+          imagePullPolicy: Always
+          securityContext:
+            runAsUser: 802
+          ports:
+            - containerPort: 9763
+              protocol: TCP
+            - containerPort: 9443
+              protocol: TCP
+      initContainers:
+        - name: wso2-is-init
+          image: busybox
+          command: ['sh', '-c', 'until nslookup wso2is-with-analytics-is-service; do echo waiting for wso2-is-service; sleep 2; done']
```

Comment:
This init container should check for **wso2is-with-analytics-is-analytics-service**, right? since that is the service which exposes analytics pods.

Reply:
done